registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -260,6 +261,7 @@
     "sha256:8a84e4e228183193c9ec73cefb5a99016d7f0797240d7c72a71ea4d6797f2ac9": ["v1.11.0-rc.0"]
     "sha256:a50b74f61ec806546fdc59ec983f00029f310fafd3f3e97ac21f04806975aab8": ["v1.11.0-rc.1"]
     "sha256:fe607f15b1a8218c632323a722ddb6bc9c4942850620deae2e63cb2c100659bc": ["v1.11.1"]
+    "sha256:47255294ac7130e8efeedb490da88d59d52734beb2f6b2d376ae1f50dc5e9867": ["v1.11.2"]
```

Comment:
SHA `47255294ac71` matches clusterctl v1.11.2 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:47255294ac7130e8efeedb490da88d59d52734beb2f6b2d376ae1f50dc5e9867/details).